aptos-move/e2e-move-tests/src/tests/aggregator_v2.data/pack/sources/events_with_aggregators.move

```diff
@@ -0,0 +1,76 @@
+module 0x1::events_with_aggregators {
+    use std::signer;
+    use aptos_framework::account;
+    use aptos_framework::aggregator_v2::{Self, Aggregator, AggregatorSnapshot};
+    use aptos_framework::event;
+
+    const E_COUNTER_ALREADY_EXISTS: u64 = 0;
+    const E_COUNTER_DOES_NOT_EXIST: u64 = 1;
+
+    struct EventV1 has drop, store {
+        value: AggregatorSnapshot<u64>,
+    }
+
+    #[event]
+    struct EventV2 has drop, store {
```

Comment:
EventV2 has #[event] but not EventV1.
What's the difference? Is EventV1 still an event without the annotation?

Reply:
EventV2 is #[event] 
EventV1 is any object, but has an onchain handle. 

you can check out EventV2 AIP for more details